src/Layout/ConnectedLayout.tsx

```diff
@@ -7,35 +7,30 @@ import {
     initializeRequestedVotes,
     initializeRequestedMetrics,
     InitializeRequestedParameters,
+    initializeRequestedPartyMap,
 } from "../requested-data";
 import { initializeComputation } from "../computation";
 import { initializeComputationMenu } from "./ComputationMenu";
 import { initializePresentation } from "./PresentationMenu";
 import { stateIsInvalid } from "../store/version";
 import { rawParametersToParametersConverter } from "../requested-data/requested-data-utilities";
 import { RootState } from "../reducers";
+import { Dictionary } from "utilities/dictionary";
```

Comment:
Codacy found an issue: [Module 'utilities' is not listed as dependency in package.json](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5947793)